salt/tests/unit/formulas/config.yaml

```diff
@@ -0,0 +1,582 @@
+---
+# Default context options are listed here.
+# Please be mindful of the number of cases generated, as these will apply to
+# many formulas.
+default_opts:
+  os:
+    - CentOS/7
+  saltenv:
+    - __default__  # Will use "metalk8s-{{ pillar.metalk8s.cluster_version }}"
+  minion_state:
+    - ready
+  volumes:
+    - none
+  mode:
+    - minion
+
+metalk8s:
+  # Use the special `_skip` keyword to omit rendering of a directory or formula
+  _skip: true
```

Comment:
I would rather have `_skip` set to false at the top level (or even as a default) and explicitly skip the things we don't want to test, but maybe it's already what is planned when everything will be ready.

Reply:
`_skip` is already set to `false` by default, but since I wanted to build this PR by progressively adding more formulas to the scope, one of the first commits "skips everything except `map.jinja`" and the following ones are iteratively "unskipping" parts of the `metalk8s/` tree.

The goal is to get rid of the `_skip` statements after this PR, so nothing is skipped.